shamedpets/app/views/offenses/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Listing offenses</h1>
+<%= link_to 'New offense', new_offense_path %>
+
+<table>
+  <tr>
+    <th>Crime</th>
+    <th>Description</th>
+    <th>Picture</th>
+  </tr>
+
+  <% @offenses.each do |offense| %>
+    <tr>
+      <td><%= offense.crime %></td>
+      <td><%= offense.description %></td>
+      <td><img src= "<%= offense.picture %>" width="250px"></img></td>
+      <td><%= link_to 'Show', offense_path(offense) %></td>
+      <td><%= link_to 'Edit', edit_offense_path(offense) %></td>
+      <td><%= link_to 'Destroy', offense_path(offense),
+              method: :delete,
+              data: { confirm: 'Are you sure?' } %></td>
+    </tr>
+  <% end %>
+</table>
```

Comment:
I think this is a good use of a table. Just be aware some developers are very opinionated against the use of tables and prefer grids of divs.

👍  tho!